tests/test_adaptor.hh

```diff
@@ -276,6 +279,140 @@ namespace rascal {
     json factory_args{};
   };
 
+  struct MultipleStructureManagerNLFixtureNoCenter {
```

Comment:
what does `NoCenter` refer to here? `CenterMask` maybe?

Reply:
it refers to the fact that some centers are omitted in contrast to other similar struct. NoCenter is how we have been calling this functionality since the begining.